scan-payment/src/main/java/com/getbouncer/scan/payment/analyzer/NameDetectAnalyzer.kt

```diff
@@ -73,7 +105,31 @@ class NameDetectAnalyzer private constructor(
         )
     }
 
-    private suspend fun processPredictions(
+    private suspend fun processExpiryPrediction(
+        expRect: RectF,
+        bitmapForObjectDetection: Bitmap
+    ): ExpiryDetect.Expiry? {
+        // Crop out the expected aspect ratio
+        val targetAspectRatio = ASPECT_RATIO
+        val scaledRect = expRect.scaled(bitmapForObjectDetection.size())
+        val scaledExpRectNewHeight = scaledRect.width() * targetAspectRatio
```

Comment:
this logic should be moved to the `ExpiryDetect` class. Encapsulate the image manipulation with the logic.